File: Eclipse/VersaoComBanco/ProjetoFinalDm102/src/br/inatel/mobile/pos/dm102/trabalhofinal/model/ClienteJuridico.java
package br.inatel.mobile.pos.dm102.trabalhofinal.model;

import br.inatel.mobile.pos.dm102.trabalhofinal.dao.ClienteJuridicoDAO;

public class ClienteJuridico extends Cliente {

	private String cnpj;
	private String razaoSocial;
	private String inscricaoEstadual;

	public ClienteJuridico(String nome, String endereco, String telefone, String cnpj, String razaoSocial,
			String inscricaoEstadual) {
		super(nome, endereco, telefone);
		this.cnpj = cnpj;
		this.razaoSocial = razaoSocial;
		this.inscricaoEstadual = inscricaoEstadual;
	}

	public String getCnpj() {
		return cnpj;
	}

	public String getRazaoSocial() {
		return razaoSocial;
	}

	public String getInscricaoEstadual() {
		return inscricaoEstadual;
	}

	public int getNumeroDeAtendimentos() {
		return ClienteJuridicoDAO.contarNumeroDeAtendimentos(this.cnpj);
	}

	@Override
	public String toString() {
		return super.toString() + " -- CNPJ: " + cnpj + " -- Razao Social: " + razaoSocial + " -- Inscricao Estadual: "
				+ inscricaoEstadual + " -- Numero de Atendimentos: " + getNumeroDeAtendimentos();
	}

}

File: VisualStudio/ProjetoFinalDm102/ProjetoFinalDm102/ClienteJuridico.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProjetoFinalDm102
{
    class ClienteJuridico : Cliente
    {
        private string cnpj;
        private string razaoSocial;
        private string inscricaoEstadual;

        public ClienteJuridico(string nome, string endereco, string telefone, string cnpj, string razaoSocial, string inscricaoEstadual) : base(nome, endereco, telefone)
        {
            this.cnpj = cnpj;
            this.razaoSocial = razaoSocial;
            this.inscricaoEstadual = inscricaoEstadual;
        }

        public string GetCnpj()
        {
            return cnpj;
        }

        public string GetRazaoSocial()
        {
            return razaoSocial;
        }

        public string GetInscricaoEstadual()
        {
            return inscricaoEstadual;
        }

        public override string ToString()
        {
            return base.ToString() + " - Cnpj: " + cnpj + " - Razao Social: " + razaoSocial + " - Inscricao Estadual: " + inscricaoEstadual;
        }
    }
}
